File: src/main/java/com/example/aeropass/entities/Passageiro.java
package com.example.aeropass.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Passageiro {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Long id;

    public String nome;

    public String cpf;

    public String passaporte;

    public String telefone;

    public String email;

    private EnumStatusPassageiro status = EnumStatusPassageiro.ATIVO;
}

File: front/app/types/passageiro.ts
export class Passageiro{
    constructor(
        public id:number | null,
        public nome:string,
        public cpf:string,
        public passaporte:number,
        public telefone :string,
        public email :string,
        public status:string
    ){}
}

export interface PassageiroFormProps{
    passageiroExistente?:Passageiro
}